src/dir.rs

```diff
@@ -129,6 +149,11 @@ impl Dir {
     /// Create file for writing (and truncate) in this directory
     ///
     /// Deprecated alias for `write_file`
+    ///
+    /// Note that this method does not resolve symlinks by default, so you may have to call
```

Comment:
This method will fail if the target path is a symlink. May also advise creating an unnamed file and then rename to a target name (replacing symlink if there are).

Reply:
Thanks for the suggestion! I assume this would apply equally to `append_file` as well?